README.rst

```diff
@@ -0,0 +1,13 @@
+Flocker Vagrant Environment
+===========================
+
+Files in this repository are used to set up Vagrant environments in Flocker's `Getting Started guide`_.
+
+Flocker is a data volume manager and multi-host Docker cluster management tool, which is being developed by `ClusterHQ`_.
+With it you can control your data using the same tools you use for your stateless applications by harnessing the power of ZFS on Linux.
```

Comment:
We're trying to get rid of the sentence about "power of ZFS" - copy phrasing from https://github.com/ClusterHQ/flocker/pull/1144/files I guess.